domain/order.py

```diff
@@ -33,9 +37,16 @@ class OrderStatus(Enum):
 
 class Order(BaseModel):
     user_id: PositiveInt
```

Comment:
We don't use raw ids in domain logic when reffer to another entity. Use the entity type instead.
```suggestion
    user: User
```